File: src/main/java/br/com/leonardoferreira/jirareport/repository/UserConfigRepository.java
package br.com.leonardoferreira.jirareport.repository;

import br.com.leonardoferreira.jirareport.domain.UserConfig;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserConfigRepository extends CrudRepository<UserConfig, Long> {

    UserConfig findByUsername(String username);

}

File: application/src/main/kotlin/br/com/jiratorio/repository/UserConfigRepository.kt
package br.com.jiratorio.repository

import br.com.jiratorio.domain.entity.UserConfig
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface UserConfigRepository : CrudRepository<UserConfig, Long> {

    fun findByUsername(username: String): UserConfig?

}
